src/RestApi/StoreApi/Controllers/CartShippingRates.php

```diff
@@ -246,6 +262,9 @@ protected function get_shipping_packages( $request ) {
 		$packages = WC()->cart->get_shipping_packages();
 
 		foreach ( $packages as $key => $package ) {
+			if ( ! $request['country'] ) {
```

Comment:
Why is this within the loop? If the country is not set, the loop over $packages won't be needed.

Reply:
Oh, right. That didn't make a lot of sense. :sweat_smile: 

Good catch, fixed in ac0f834f5e843ae035dfef8206f0b4eac4d86dfd.